src/main/java/DBot.java

```diff
@@ -0,0 +1,111 @@
+import java.util.*;
+
+public class DBot {
+    private static boolean isOn;
+    private static List<Task> datas;
+
+    public static void main(String[] args) {
+        isOn = true;
+        datas = new ArrayList<>();
```

Comment:
Perhaps you could use a more descriptive name for your array list, as it is unclear what is being stored here.